client/components/data/query-billing-transaction/README.md

```diff
@@ -0,0 +1,35 @@
+Query Billing Transaction
+================
+
+`<QueryBillingTransaction transactionId={ transactionId } />` is a React component used to ensure that a single transaction is fetched and ready to display. It also renders a `<QueryBillingTransactions />` component to first check if the requested `transactionId` is included in the most recent and cached transactions.
```

Comment:
We'll need to update/remove this section if we decide to remove the nested query component.

Reply:
If you feel it's necessary, you can add a note that usage with `<QueryBillingTransactions />` is recommended, although this feels a little anti-pattern-y in the first place.